clamav/client.py

```diff
@@ -27,9 +27,7 @@
 )
 
 
-class ClamAVApi(object):
-    '''Implements ClamAV REST API functionality.
-    '''
+class ClamAVClient(object):
```

Comment:
please push this commit to master. or sqash into appropriate commit from this PR